lib/reqTemplate.js

```diff
@@ -138,7 +140,7 @@ function splitAndPrepareTAsseblyTemplate(templateSpec, options) {
 
                 // TODO: Enable single brace structures outside of URI once
                 // the specs are migrated to double brace.
-                if (!inDoubleBrace && options.isURI) {
+                if (!inDoubleBrace) {
                     if (/^\+/.test(currentTemplate)) {
```

Comment:
Double braces are a custom extension and not part of the URI template spec. This means that we can't use them in public path templates. I think there is value in keeping path templates and request handler templates consistent, so I would recommend to keep `{+path}` working.